### ISSUE TITLE
How can I connect this to my own app?!

### ISSUE THREAD

**Issue by derwaldgeist** [thebakeryio/meteor-ddp-monitor#67]
Maybe I'm just too dumb, but I can't get this working with my own app. If I install the extensions from the Chrome store, it seems to show some other app, with different templates and such. But there's no user interface to point it to my own app. How exactly can I set-up this extension for my own app?!

**Comment by btorresgil** [comment]
Yeah, I’m confused too. It seems to show demo data, not the data for the App I’m viewing.
